src/Phinx/Db/Adapter/PostgresAdapter.php

```diff
@@ -875,12 +875,14 @@ public function dropDatabase($name)
      * @param  mixed $default
      * @return string
```

Comment:
Needs adding to the docblock

Reply:
For return statements? I thought only params do.

Reply:
A new param was added

Reply:
fixed docblock